.travis.yml

```diff
@@ -1,18 +1,22 @@
 language: go
 
 go:
-  - "1.12"
+  - "1.13"
+  - "1.14"
 
 go_import_path: github.com/kubeflow/tf-operator
 
 install:
   # get coveralls.io support
+  - export GO111MODULE=on
   - go get github.com/mattn/goveralls
   - curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.18.0
-  - go install ./vendor/k8s.io/code-generator/cmd/openapi-gen
+  # - go install ./vendor/k8s.io/code-generator/cmd/openapi-gen
+  - go mod vendor
 
 script:
-  - hack/verify-codegen.sh
```

Comment:
I think we should keep the verify script.

Reply:
Sure, will add it back later after the code-gen related dependencies get fixed.

Reply:
OK, thanks for the PR